src/store.js

```diff
@@ -1,28 +1,43 @@
 import thunkMiddleware from "redux-thunk";
 import { createStore, combineReducers, applyMiddleware, compose } from "redux";
-import { loadLogin, removeLogin } from "./localStorage";
+import { loadLogin, removeState, loadTeams } from "./localStorage";
 import { authentication } from "src/modules/login/reducer";
 import { goal } from "src/modules/goal/reducer";
 import { feed } from "src/modules/feed/reducer";
 import { transaction } from "./modules/transaction/reducer";
 import { profile } from "./modules/profile/reducer";
 import { statistics } from "./modules/statistics/reducer";
+import { teams } from "./modules/teams/reducer";
 
 const rootReducer = combineReducers({
   authentication,
   goal,
   feed,
   transaction,
   profile,
-  statistics
+  statistics,
+  teams
 });
 const middleware = applyMiddleware(thunkMiddleware);
 
-const persistedLogin = loadLogin();
+const persistedStateLogin = loadLogin();
+const persistedStateTeams = loadTeams();
+
+const givePersistedState = () => {
```

Comment:
Can you explain me what this function does? How can it be that when you expect `persistedStateLogin` to be false, you check again if `persistedStateLogin` is true?